packages/p2p/src/components/buy-sell/buy-sell-modal.jsx

```diff
@@ -36,12 +37,18 @@ BuySellModalFooter.propTypes = {
 
 const BuySellModal = observer(({ table_type, selected_ad, should_show_popup, setShouldShowPopup }) => {
     const isMounted = useIsMounted();
-    const { general_store, order_store } = useStores();
+    const { buy_sell_store, general_store, order_store } = useStores();
+    const { local_currency } = buy_sell_store?.advert || {};
     const submitForm = React.useRef(() => {});
     const [error_message, setErrorMessage] = React.useState(null);
     const [is_submit_disabled, setIsSubmitDisabled] = React.useState(true);
-    const [page_footer_parent, setPageFooterParent] = React.useState(null);
-
+    const [page_footer_parent, setPageFooterParent] = React.useState(
+        <BuySellFormReceiveAmount
+            is_sell_advert={buy_sell_store.is_sell_advert}
+            local_currency={local_currency || ''}
```

Comment:
We shouldn't send an empty string here.

Reply:
as local_currency is required we have a warning on console that it shouldn't be undefined, but after the user clicks on buy/sell button it receives a currency as a value

Reply:
should remove it anyway?